<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lookup</class>
 <widget class="QDialog" name="lookup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>look-up</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/icon.png</normaloff>icons/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTextEdit" name="txtResultLookup"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
